tools/Az.Tools.Installer/Az.Tools.Installer.psd1

```diff
@@ -55,7 +55,7 @@
     
     # Assemblies that must be loaded prior to importing this module
     RequiredAssemblies = @(
-        '.\assembly\Microsoft.ApplicationInsights.2.12.0\Microsoft.ApplicationInsights.dll'
+        '.\assembly\Microsoft.ApplicationInsights.2.21.0\Microsoft.ApplicationInsights.dll'
```

Comment:
Let me know if this should **not** be updated (I see that it failed some tests in ADO, but I can't see where the relevant test is specified)

Reply:
> Let me know if this should **not** be updated (I see that it failed some tests in ADO, but I can't see where the relevant test is specified)
Hi @matsest, Welcome to contribute to Az.Tools.Installer！

The pipeline fails of other reasons. We are fixing it.
However, we still cannot upgrade ApplicationInsights to the latest version for PowerShell 7.0+ cannot support it. I have change the version to 2.13.1.

I also revert the change of `Repository` for we keep the parameter as optional to be consistent with PowerShellGet. To clarify the ambiguity, I add more explanation in the help message.

I remove the Az.Tools.Installer-help.xml for we will generate it every time we release a new version.

Please review the change I've made.

Reply:
@matsest Could you report the issue here https://github.com/Azure/azure-powershell/issues？ Thanks

Reply:
@msJinLei I would assume the issue already listed in the description of this PR is covering this sufficiently - see https://github.com/Azure/azure-powershell/issues/19839

Let me know if there should be created another one

Reply:
OK. I will check it later.

Reply:
@msJinLei I see this was merged but the issue regarding the Repository parameter was not resolved as I suggested.  Https://github.com/Azure/azure-powershell/issues/19839 should be reopened..

Reply:
> @msJinLei I see this was merged but the issue regarding the Repository parameter was not resolved as I suggested. [Https://github.com/Azure/azure-powershell/issues/19839](Https://github.com/Azure/azure-powershell/issues/19839) should be reopened..

I created an issue for it https://github.com/Azure/azure-powershell/issues/20209 and will fix it soon. Thanks for reporting.